example/config/webSocket.js

```diff
@@ -0,0 +1,36 @@
+const Web3 = require("web3");
+const EEAClient = require("../../src");
+
+// Define a WebSocket provider, passing in the desired options:
+// See https://github.com/ethereum/web3.js/tree/1.x/packages/web3-providers-ws for options
+const providerOptions = {
+  timeout: 30000, // ms
+  headers: {
+    authorization: "Basic username:password"
```

Comment:
It would be good if this example had what Besu will be expecting when checking if the request is authenticated, the Bearer token.

`authorization: "Bearer ewogICJhbGciOiAibm9uZSIsCiAgInR5cCI6ICJKV1QiCn0.eyJpYXQiOjE1MTYyMzkwMjIsImV4cCI6NDcyOTM2MzIwMCwicGVybWlzc2lvbnMiOlsibmV0OnBlZXJDb3VudCJdfQ"`